frontend/src/components/explorer/mapViewer/Svgmap.vue

```diff
@@ -330,10 +331,12 @@ export default {
     },
     applyLevelsOnMap() {
       if (Object.keys(this.computedLevels).length === 0) {
-        $(`#svg-wrapper .${this.componentClassName} .shape`).attr('fill', this.componentDefaultColor);
+        Object.values(DATA_TYPES_COMPONENTS).forEach((dataType) => {
```

Comment:
It might be because of this comment I left on Slack:
> A user should be able to overlay multiple data types at the same time, so they should not be reset.

Reply:
Before PR #676, this section looked like this

```
    applyHPARNAlevelsOnMap(RNAlevels) {
      this.HPARNAlevels = RNAlevels;
      if (Object.keys(this.HPARNAlevels).length === 0) {
        $('#svg-wrapper .enz .shape').attr('fill', this.defaultGeneColor);
```

I think @e0 adapted it to make in applicable to more data sources. However, when switching between sources/setting data set to None, the coloring was not reset entirely on the map. The code you commented fixes this. Please play around in the data overlay section if you want to see how it looks like with and without this fix